packages/core/src/Reputation.ts

```diff
@@ -0,0 +1,77 @@
+import assert from 'assert'
+import { BigNumber, BigNumberish } from 'ethers'
+import { hash5, hashOne } from '@unirep/crypto'
+
+import { IReputation } from './interfaces'
+
+export default class Reputation implements IReputation {
+    public posRep: BigNumber
```

Comment:
If we don't modify value,  why we don't use readonly.

Reply:
it could be updated